### PR TITLE
Another round to fix xml tags

### PR DESCRIPTION
**Pull request by BethanyZhou**

### CODE REVIEW

File: powershell/cmdlets/class.ts [hunk 1]
```diff
@@ -1731,3 +1732,12 @@ export class CmdletClass extends Class {
...
 }
+function GetParameterReferenceString(typeName: string): string {
+  if (typeName.endsWith("[]")) {
+    return `<see cref="${typeName.substring(0, typeName.lastIndexOf("[]"))}">${typeName}</see>`;
+  } else if (typeName.endsWith("?")) {
+    return `<see cref="${typeName.substring(0, typeName.lastIndexOf("?"))}">${typeName}</see>`;
+  }
+  return `<see cref="${typeName}"/>`;
+}
+
```
Comment (BethanyZhou) [inline powershell/cmdlets/class.ts:1743]: Is there any advice? I prefer regex here. But I'm unfamiliar with typescript.

Comment (dolauli) [reply powershell/cmdlets/class.ts:1743]: How about to replace the GetParameterReferenceString function with following one line code.

<see cref="${each.language.csharp.responseType.replace(/\[|\]|\?/g, '')}">${each.language.csharp.responseType}</see>

Comment (dolauli) [reply powershell/cmdlets/class.ts:1743]: Should be 
```
<see cref="${each.language.csharp.responseType.replace(/\[|\]|\?/g, '')}">${each.language.csharp.responseType}</see>
```

Comment (BethanyZhou) [reply powershell/cmdlets/class.ts:1735]: Thank you!

**Review by dolauli — APPROVED** [review]
LGTM

**Pull request merged**